R/Rules-methods.R

```diff
@@ -3135,7 +3135,7 @@ setMethod("stopTrial",
       prob_target <- stopping@prob_target
 
       ## checks
-      stopifnot(is.probability(prob_target))
+      stopifnot(test_probability(prob_target))
```

Comment:
@danielinteractive -> as above should I use instead assert_probability(prob_target), to avoid the stopifnot?

Reply:
Updates completed, passed all checks